src/consensus/pbft/libbyz/message.cpp

```diff
@@ -80,9 +80,11 @@ void Message::set_size(int size)
   CCF_ASSERT(msg && ALIGNED(msg), "Invalid state");
   if (!(max_size < 0 || ALIGNED_SIZE(size) <= max_size))
   {
-    LOG_INFO << "Error - size:" << size
-             << ", aligned_size:" << ALIGNED_SIZE(size)
-             << ", max_size:" << max_size << std::endl;
+    LOG_INFO_FMT(
```

Comment:
I'm a bit confused by this, should that be part of the assert message on the next line instead?

Reply:
ccf_assert currently cannot take a format string. #1246 tracks this.

Reply:
Ah fair enough. It could still take a `fmt::format("something something")`, but building it in is probably better anyway.